src/ui/cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
Can we clean up the boilerplate comments in here and focus it just on the pathways we are going to use and support?